src/shared/format-table.h

```diff
@@ -35,6 +35,8 @@ typedef enum TableDataType {
         TABLE_IFINDEX,
         TABLE_IN_ADDR,  /* Takes a union in_addr_union (or a struct in_addr) */
         TABLE_IN6_ADDR, /* Takes a union in_addr_union (or a struct in6_addr) */
+        TABLE_SD_128_T,
```

Comment:
type is called sd_id128_t btw, i.e. the "id" got lost?

Reply:
I'll rebase on top of your PR, and then this will go away.